app.js

```diff
@@ -16,7 +17,9 @@ const {
 
 app.use(express.json());
 
-app.get('/api', getEndpoints);
+app.get('/api', (req, res) => {
+  res.status(200).send({ endpoints });
+});
```

Comment:
I don't know whats caused this change but it sill should really be separated out into a controller.